feedback from Med's IESG review of composite-sigs (#347)
lamps-wg/draft-composite-sigs: draft-ietf-lamps-pq-composite-sigs.md

    multi-algorithm scheme, such that the resulting composite
    cryptographic element is exposed as a singular interface
    of the same type as the component cryptographic elements.
    For example this could be an asymmetric algorithm such as
    "ML-DSA-65" or "RSASSA-PSS".

**DER**:
  Distinguished Encoding Rules as defined in [X.690].

**ECDSA**: The Elliptic Curve Digital Signature Algorithm defined in [FIPS.186-5].

**EdDSA, Ed25519 and Ed448**: Edwards-curve Digital Signature Algorithm (EdDSA)
  defined in [RFC8410] with two parameter sets: Ed25519 and Ed448.

**ML-DSA**: The Module-Lattice-Based Digital Signature Standard defined in [FIPS.204].

**PKI**:
  Public Key Infrastructure, as defined in [RFC5280].

**POST-QUANTUM TRADITIONAL (PQ/T) HYBRID SCHEME**:
  {{RFC9794}} defines a PQ/T Hybrid Scheme as:
  A multi-algorithm scheme where at least one component algorithm
  is a post-quantum algorithm and at least one is a traditional algorithm.

**PROTOCOL BACKWARDS COMPATIBILITY**:
  A property whereby a new feature
  can be added to a protocol without requiring any changes to the
  protocol's specification and only minimal changes to its

>   implementations (such as adding new identifiers).

Author: ```suggestion
  implementations.
```

In order to ensure fresh keys, the key generation functions MUST be executed for both component algorithms. Compliant parties MUST NOT use, import or export component keys that are used in other contexts, combinations, or by themselves as keys for standalone algorithm use. For more details on the security considerations around key reuse, see {{sec-cons-key-reuse}}.

> If one of the component `KeyGen()` routines returns an error, then this error MUST be propagated by the `Composite-ML-DSA.KeyGen()` routine.

Author: ```suggestion
If one of the component `KeyGen()` routines returns an error, then the  `Composite-ML-DSA.KeyGen()` routine MUST also return an error.
```

       input into this function.
~~~

## Interoperability of legacy algorithms

> The legacy component algorithms, particularly RSA and ECDSA can themselves have interoperability issues which will propagate to become interoperability issues in the composite. For example, this specification RECOMMENDS an RSA expenent of 65537, but other values are possible. Similarly, due to the details of DER encoding, keys that happen to have leading zeros could appear to be smaller than the required key size even though they are actually acceptable.

Author: ```suggestion
The legacy component algorithms, particularly RSA and ECDSA can themselves have interoperability issues which will propagate to become interoperability issues in the composite. For example, this specification RECOMMENDS an RSA exponent of 65537, but other values are possible. 
```
